carbon-design-system/carbon-for-ibm-dotcom
# fix(masthead): ensure custom L0 nav items work

### Related Ticket(s)

none

### Description

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/11420 ensures support for customizing the L0 nav items, but it doesn't account for the `navLinks` property. This PR adds that support, and it also includes a test to make sure L0 customization is working as expected.

## To test

1. Visit this Codepen example: https://codepen.io/jakaeser/pen/LYavrKK
2. See that the custom L0 nav items are not applying.
3. Comment out the v2 latest CDN <script>, then uncomment the deploy preview CDN <script>
4. Verify the custom L0 nav items apply.
5. In the Pen's JS, replace the `l0Data` property with `navLinks`. Verify the custom L0 items still apply.

### Changelog

**New**

- Adds an e2e test for customizing L0 items.

**Changed**

- Supports `navLinks` property when using `<c4d-masthead-container>` and customizing L0 items.
- Corrects the prefixes used in masthead e2e tests.
